pytest/tests/sanity/memtrie_disktrie_switch.py

```diff
@@ -244,38 +251,39 @@ def __deploy_contracts(self):
 
     def __create_accounts(self):
         """Creates the test accounts.
-        
+
         Waits for the create-account transactions to complete."""
+
+        # Update nonces for the validator accounts, as we will use them to create the new accounts.
+        self.__retrieve_nonces([node.signer_key for node in self.nodes])
+
+        create_account_tx_list = []
         account_keys = []
-        for account_id in ALL_ACCOUNTS:
+        for i in range(len(ALL_ACCOUNTS_PREFIXES)):
+            # Choose a validator node to sign the transaction for creating the new account.
```

Comment:
One of the nodes in this test is a non-validator

Reply:
updated comment.